manifests/0000_50_cluster-platform-operator-manager_07-aggregated-clusteroperator.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   annotations:
     include.release.openshift.io/ibm-cloud-managed: "true"
     include.release.openshift.io/self-managed-high-availability: "true"
-    release.openshift.io/feature-gate: TechPreviewNoUpgrade
+    release.openshift.io/feature-set: TechPreviewNoUpgrade
   name: platform-operators-aggregated
   namespace: openshift-platform-operators
```

Comment:
ClusterOperator [are cluster-scoped resources][1], so this line is not necessary.

[1]: https://github.com/openshift/api/blob/cce310ad2932f6de24491052d506926e484c082c/config/v1/types_cluster_operator.go#L9